### PR TITLE
feat: added typography component

### PR DESCRIPTION
**Pull request by GuptaEesh**
Various types of typographies were added:

>Font-Weight
Font-Sizes
Font-Styles
Alignment
Color

typo

### CODE REVIEW

**Review by theprogrammerinyou — APPROVED** [review]
Code looks good to me. Good job.
1. Good looking UI.
2. Liked that folder structure.
3. Good usage of code breaking and splitting it into components.

File: components/typography.css [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+.text-strike {
+    text-decoration: line-through;
+}
+
+.text-underline {
+    text-decoration: underline;
+}
+
+
```
Comment (saadahmadhere) [inline components/typography.css:76]: Here is an extra line space.

File: components/typography.css [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+.size-28 {
+    font-size: var(--size-28);
+}
+
+.size-32 {
+    font-size: var(--size-32);
+}
+
+
```
Comment (saadahmadhere) [inline components/typography.css:65]: Extra line space.

File: components/typography.css [hunk 3]
```diff
@@ -0,0 +1,98 @@
+/* Font - Weight*/
+.eg-typo{
+    display: flex;
+    flex-direction: column;
+    gap: var(--size-16);
+}
+.light {
```
Comment (saadahmadhere) [inline components/typography.css:8]: need a line space before this.

**Review by saadahmadhere — COMMENTED** [review]
To the point use of variables. Just need a double check on the line spacing.

File: components/typography.css (hunk 3 above)
Comment (GuptaEesh) [reply components/typography.css:8]: This is automated spacing.

File: components/typography.css (hunk 1 above)
Comment (GuptaEesh) [reply components/typography.css:76]: rectified.

File: components/typography.css (hunk 2 above)
Comment (GuptaEesh) [reply components/typography.css:65]: rectified.

**Comment by GuptaEesh** [comment]
Done with spacing. Merging it now.

**Pull request merged**